bitrise.yml

```diff
@@ -88,23 +88,6 @@ workflows:
             cp Carthage/Build/iOS/MozillaAppServices.framework/sdk_generator.sh ./
   3_provisioning_and_npm_installation:
     steps:
-    - script@1:
```

Comment:
@nbhasin2 I'm not missing anything that will be broken by removing this right?

Reply:
@isabelrios I am not too familiar as to why we are setting provisional profile, we can remove this step considering profiles are set manually. If we encounter any errors we can add back the profiles that aren't working :)